responders/src/UI/embc-responder/src/app/feature-components/wizard/wizard.component.ts

```diff
@@ -103,14 +103,19 @@ export class WizardComponent implements OnInit, OnDestroy {
     this.dialog
       .open(DialogComponent, {
         data: {
-          component: ExitWizardDialogComponent
+          component: YesNoDialogComponent,
+          text: 'Are you sure you want to exit the wizard?',
+          text2:
```

Comment:
Create dialog object to set these values. No static value should be in the components. Add all static values to global constants